scootapi/client/run_job_cmd.go

```diff
@@ -33,13 +33,15 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 
 // Types to handle JobDefinitions from JSON files
 type JobDef struct {
-	Tasks map[string]TaskDef
+	Tasks     map[string]TaskDef
+	TimeoutMs int32
```

Comment:
Does that conform to expected behavior? Job timeout should either be called defaultTaskTimeout in this case or it should be used to limit total job time imo.